_layouts/default.html

```diff
@@ -1,16 +1,6 @@
 <!DOCTYPE html>
 <html lang="{{ site.lang | default: "en-US" }}">
   <head>
-
-    {% if site.google_analytics %}
-      <script async src="https://www.googletagmanager.com/gtag/js?id={{ site.google_analytics }}"></script>
-      <script>
-        window.dataLayer = window.dataLayer || [];
-        function gtag(){dataLayer.push(arguments);}
-        gtag('js', new Date());
-        gtag('config', '{{ site.google_analytics }}');
-      </script>
-    {% endif %}
```

Comment:
If you prefer the gtag code, add a `_includes/head-custom-google-analytics.html` file to your repo with the gtag code! Unfortunately this reverts https://github.com/pages-themes/cayman/pull/49 due to the backwards-incompatible change that it introduced for users of "classic" v3 GA.